zml/tokenizer.zig

```diff
@@ -306,6 +372,11 @@ pub const Tokenizer = struct {
                 if (output.items.len > 0) try output.append(' ');
             }
 
+            var tmp_buffer: [100]u8 = undefined;
```

Comment:
tmp_buff is way too big, but I don't think it's needed if you properly initialize the tokenizer